pkg/kv/kvprober/kvprober_integration_test.go

```diff
@@ -92,6 +94,55 @@ func TestProberDoesReadsAndWrites(t *testing.T) {
 		require.Zero(t, p.Metrics().ProbePlanFailures.Count())
 	})
 
+	t.Run("happy path with tracing enabled", func(t *testing.T) {
```

Comment:
I think you can just add the new logic to the existing test with case name "happy path".

Reply:
Done!